server/views/pages/licence/AP.test.ts

```diff
@@ -38,7 +38,7 @@ describe('Print an AP licence', () => {
         ],
         bespokeConditions: [{ text: 'Bespoke condition 1' }],
       },
-      additionalConditions: [
```

Comment:
Not a great fan of this test.. not sure its testing anything, other than content. No logic being tested. I left this test in but did not add tests for PSS and AP_PSS licences

Reply:
It checks that the document structure given in data is rendered into the sections / counts expected, but largely repeating what the integration test does as well.